packs/howl-of-the-wild-bestiary/sunscale-serpent.json

```diff
@@ -288,7 +288,87 @@
                     "remaster": false,
                     "title": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "domain": "all",
+                        "key": "RollOption",
+                        "option": "bond-with-mortal",
+                        "toggleable": true
+                    },
+                    {
+                        "key": "FlatModifier",
+                        "predicate": [
+                            "bond-with-mortal"
+                        ],
+                        "selector": "hp",
+                        "value": 28
+                    },
+                    {
+                        "key": "FlatModifier",
+                        "predicate": [
+                            "bond-with-mortal"
+                        ],
+                        "selector": [
+                            "attack-roll",
+                            "damage"
+                        ],
+                        "type": "status",
+                        "value": 2
+                    },
+                    {
+                        "add": [
+                            "incorporeal",
+                            "spirit"
+                        ],
+                        "key": "ActorTraits",
+                        "predicate": [
+                            {
+                                "not": "bond-with-mortal"
+                            }
+                        ]
+                    },
+                    {
+                        "key": "Immunity",
+                        "mode": "remove",
+                        "predicate": [
+                            "bond-with-mortal"
+                        ],
+                        "type": [
+                            "disease",
+                            "paralyzed",
+                            "poison"
+                        ]
+                    },
+                    {
+                        "key": "Resistance",
+                        "mode": "remove",
+                        "predicate": [
+                            "bond-with-mortal"
+                        ],
+                        "type": "all-damage",
+                        "value": 0
+                    },
+                    {
+                        "key": "DamageDice",
+                        "override": {
+                            "damageType": "piercing"
+                        },
+                        "predicate": [
+                            "bond-with-mortal"
+                        ],
+                        "selector": "jaws-damage"
+                    },
+                    {
+                        "key": "DamageDice",
+                        "override": {
+                            "damageType": "slashing"
```

Comment:
Wasn't sure if this should be bludgeoning, but the ability says "typically piercing or slashing"